test/find.test.js

```diff
@@ -179,6 +179,21 @@ describe('find', () => {
           name: 'aleph',
         },
       ]),
+      t.db
+        .collection('test_null_values')
+        .insert(
+          [
+            undefined,
+            undefined,
+            undefined,
+            null,
+            null,
+            'Alice',
+            'Bob',
+            'alpha',
+            'bravo',
+          ].map((name, i) => (name === undefined ? { _id: i + 1 } : { _id: i + 1, name }))
```

Comment:
```suggestion
          ].map((name, i) => (name === undefined ? { _id: i + 1 } : { _id: i + 1, name }))
          // Ids starts with 1 because Mongoist driver doesn't support _id as 0
```